### ISSUE TITLE
Stuck at dokku version 0.19.13

### ISSUE THREAD

**Issue by lggwettmann** [dokku/dokku#3994]
## Description of problem
I try to update to the newest Dokku version, but Dokku stays at version 0.19.13. I tried the manual installation and with dokku-update.

### Steps to Reproduce

1. I tried it with 'dokku-update' and get this:
```
root@Server:~# dokku-update
-----> Running system updates
-----> Updating all plugins
       Updating apt
Plugin (apt) updated
-----> Priming bash-completion cache
       Updating letsencrypt
root@Server:~# 
```
So no errors, but still the same version.
2. I tried it with these instructions http://dokku.viewdocs.io/dokku~v0.20.4/getting-started/upgrading/#upgrade-instructions and don't get any messages at all.

Is there some kind of cache I need to empty or can I force a specific version somehow?

**Comment by josegonzalez** [comment]
Can you include *all* of the information we ask for in our [issue template](https://github.com/dokku/dokku/blob/master/ISSUE_TEMPLATE.md)? It is helpful for debugging problems. Thanks.

**Comment by josegonzalez** [comment]
Closing as there hasn't been any update. Note that we dropped support for older Ubuntu/Debian releases that are no longer updated upstream [in 0.20.0](http://dokku.viewdocs.io/dokku~v0.20.4/appendices/0.20.0-migration-guide/#remove-support-for-unsupported-operating-systems), and highly recommend that our users upgrade their systems as necessary.

The packages _may_ still work - we build generic ones for Debian systems - but there is no guarantee that we won't depend on something that is only available on newer systems.
